source/index.md

```diff
@@ -80,6 +80,6 @@ Be sure you have added `node_modules` to your Sass's [load_paths](http://stackov
 ## What's next
 
 - [Documentations](docs): Design philosophy and tutorial
-- [Reference](reference): All the Mobi.css styles in one page

```

Comment:
Yeah I guess it makes more sense in this context.

Could change it to something like 'See the Mobi.css styles in action' or 'Examples of the Mobi.css styles' or a combination.

Something that directs the scanning eye of developers who want to see examples fast. I hate having to hunt for snippets.